pyproject.toml

```diff
@@ -32,7 +32,7 @@ dependencies = [
     # you modify these, make sure to change the corresponding line there.
     "click >= 7.0",
     "cloudpickle >= 1.1.1",
-    "fsspec >= 0.6.0",
+    "fsspec >= 0.7.4",
```

Comment:
Need fsspec>=0.7.4 (just under three-years old) to use `url_to_fs`. Not sure if I found all the places this pin needs to be changed.